eap-xp/runtime-image/image.yaml

```diff
@@ -0,0 +1,74 @@
+schema_version: 1
+
+name: "jboss-eap-7/eap-xp1-openjdk11-runtime-openshift-rhel8"
+description: "The JBoss EAP XP 1.0 openjdk11 runtime image"
```

Comment:
let's make it consistent with the builder image and use "Red Hat JBoss Enterprise Application Platform XP 1.0 OpenShift runtime image"

Reply:
In this case I also added " ... with OpenJDK X" to both as well, so they both have slightly better descriptions